demo/src/sample/styling/data.json

```diff
@@ -24,7 +24,7 @@
         "type": "str",
         "id": 43,
         "value": "David is cool",
-        "style": "highlight"
+        "style": ["highlight"]
```

Comment:
While you have identified the correct problem here, I'd like you to actually modify the type definition to allow for a bare string as a possibility for the `Styles` data type. Note that you'll also need to modify the `style` attribute parsing in `MemoryModel.drawAll` to handle the case where `obj.style` is a string. (This is correctly handled in the loop when `obj.style` is an array, so you can copy the logic from that.)

This is a separate bug fix from the `font-size` issue.